AEPMessaging/Sources/LiveActivity/AEPLiveActivityAttributes.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ Copyright 2025 Adobe. All rights reserved.
+ This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License. You may obtain a copy
+ of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software distributed under
+ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ OF ANY KIND, either express or implied. See the License for the specific language
+ governing permissions and limitations under the License.
+ */
+
+import ActivityKit
+
+/// A protocol that enables Live Activities to integrate with Adobe Experience Platform.
+///
+/// Conforming types can associate required Adobe Experience Platform data with iOS Live Activities.
+/// Any custom `ActivityAttributes` struct must implement this protocol when registering
+/// a Live Activity with the SDK.
+@available(iOS 16.1, *)
+public protocol AEPLiveActivityAttributes: ActivityAttributes {
```

Comment:
we shouldn't be prefixing our classes with `AEP` - that's more of an Obj-c convention where all classes are in the same namespace and could collide.  

maybe a better name would be `AdobeActivityAttributes`?

Reply:
also, do we need to expose this for obj-c?

Reply:
The [ActivityAttributes](https://developer.apple.com/documentation/activitykit/activityattributes) protocol is written in swift

We get error
@objc protocol 'AEPLiveActivityAttributes' cannot refine non-@objc protocol 'ActivityAttributes'


And I read ObjectiveC does not support Swift protocol with associated types  and Cannot conform to Swift protocols that require struct conformance

Therefore, to implement these live activity classes, it is necessary for customers to use Swift.